src/components/FlawForm/FlawForm.vue

```diff
@@ -257,6 +258,7 @@ const createdDate = computed(() => {
               :error="errors.source"
               :options-hidden="hiddenSources"
             />
+            <FlawLabelsTable v-if="mode==='edit'" v-model="flaw.labels!" />
```

Comment:
```suggestion
            <FlawLabelsTable v-if="flaw.uuid" v-model="flaw.labels!" />
```

Think we could shift away from using the `mode` prop